scripts/SANS/sans/algorithm_detail/move_workspaces.py

```diff
@@ -103,11 +103,6 @@ def move_backstop_monitor(ws, move_info, monitor_offset, monitor_spectrum_number
     :param monitor_offset: Offset to shift by (m)
     :param monitor_spectrum_number: The spectrum number of the monitor to shift
     """
-    # Back out early so we don't shift the monitor from the IDF position to the
-    # rear detector position by accident
-    if monitor_offset == 0.0:
-        return
-
     monitor_spectrum_number_as_string = str(monitor_spectrum_number)
     # TODO we should pass the detector ID through, not the spec num
```

Comment:
I know this isn't part of this PR but is there an issue to bring this in line with `Instrument 2.0`?

Reply:
Thanks for pointing it out anyway, we have #27024